cfme/fixtures/multi_tenancy.py

```diff
@@ -0,0 +1,67 @@
+"""
+Fixtures for Multi Tenancy
+"""
+import fauxfactory
+import pytest
+
+from cfme.base.credential import Credential
+from cfme.utils.update import update
+
+
+def new_user(appliance, groups, name=None, credential=None):
```

Comment:
_Q_ I would expect this to be a fixture, is there a reason it isn't?

Reply:
Updated it to be a fixture.

Reply:
We have so many of these fixtures defined in multiple test modules, it would be great to consolidate them, as many of them repeat the same kwargs.

Reply:
Yes, it would be nice to consolidate the fixtures. I will work on it in a follow up PR.